File: doc/design/005-dkms/how-tos/rotate-key/java/step4.java
		System.out.println("\n13. Reading new Verkey from wallet\n");
		String trustAnchorVerkeyFromWallet = Did.keyForLocalDid(walletHandle, trustAnchorDID).get();

		System.out.println("\n14. Building GET_NYM request to get Trust Anchor from Verkey\n");
		String getNymRequest = buildGetNymRequest(trustAnchorDID, trustAnchorDID).get();
		System.out.println("GET_NYM request:\n" + getNymRequest);

		System.out.println("\n15. Sending GET_NYM request to ledger\n");
		String getNymResponse = submitRequest(pool, getNymRequest).get();
		System.out.println("GET_NYM response:\n" + getNymResponse);

		System.out.println("\n16. Comparing Trust Anchor verkeys\n");
		System.out.println("Written by Steward: " + trustAnchorDID);
		System.out.println("Current from wallet: " + trustAnchorVerkeyFromWallet);
		String responseData = new JSONObject(getNymResponse).getJSONObject("result").getString("data");
		String trustAnchorVerkeyFromLedger = new JSONObject(responseData).getString("verkey");
		System.out.println("Current from ledger: " + trustAnchorVerkeyFromLedger);
		boolean match = !trustAnchorDID.equals(trustAnchorVerkeyFromWallet) && trustAnchorVerkeyFromWallet.equals(trustAnchorVerkeyFromWallet);
		System.out.println("Matching: " + match);

        // Do some cleanup.
		System.out.println("\n17. Close and delete wallet\n");
		walletHandle.closeWallet().get();
		Wallet.deleteWallet(walletName, null).get();

		System.out.println("\n18. Close pool\n");
		pool.closePoolLedger().get();

		System.out.println("\n19. Delete pool ledger config\n");
		Pool.deletePoolLedgerConfig(poolName).get();

File: docs/how-tos/rotate-key/rust/src/step4.rs
// PART 3
println!("13. Reading new Verkey from wallet");
let trustee_verkey_from_wallet = did::key_for_local_did(wallet_handle, &trustee_did).wait().unwrap();

println!("14. Building GET_NYM request to get Trust Anchor from Verkey");
let refresh_build_nym_request: String = ledger::build_get_nym_request(None, &trustee_did).wait().unwrap();

println!("15. Sending GET_NYM request to ledger");
let refresh_build_nym_response: String = ledger::submit_request(pool_handle, &refresh_build_nym_request).wait().unwrap();

println!("16. Comparing Trust Anchor verkeys");
let refresh_json: Value = serde_json::from_str(&refresh_build_nym_response).unwrap();
let refresh_data: Value = serde_json::from_str(refresh_json["result"]["data"].as_str().unwrap()).unwrap();
let trustee_verkey_from_ledger = refresh_data["verkey"].as_str().unwrap();
println!("    Written by Steward: {}", &trustee_verkey);
println!("    Current from wallet: {}", &trustee_verkey_from_wallet);
println!("    Current from ledger: {}", &trustee_verkey_from_ledger);
assert_ne!(trustee_verkey, trustee_verkey_from_ledger, "Verkey's are matched");
assert_eq!(trustee_verkey_from_wallet, trustee_verkey_from_ledger, "Verkey's did not match as expected");

// CLEAN UP
println!("17. Close and delete wallet");
wallet::close_wallet(wallet_handle).wait().unwrap();
wallet::delete_wallet(&config, USEFUL_CREDENTIALS).wait().unwrap();

// Close pool
println!("    Close pool and delete pool ledger config");
pool::close_pool_ledger(pool_handle).wait().unwrap();
pool::delete_pool_ledger(&pool_name).wait().unwrap();
